.github/workflows/build.yml

```diff
@@ -102,6 +102,10 @@ jobs:
         if: runner.os == 'macOS'
         uses: ./.github/actions/buildcocoasdk
 
+      # Only switch to newer xcode after building the Cocoa SDK so that it can keep IPHONEOS_DEPLOYMENT_TARGET=11.0
```

Comment:
That already looks annoying.

Reply:
I know, not sure what to do about that but I didn't want to unnecessarily increase the minimum deployment target we build the cocoa bindings. 

Maybe it's OK to do it, I don't really know, just didn't want to do it because it wasn't _really_ necessary, strictly speaking